### DISCUSSION TITLE
Does Stash work on Android?

### DISCUSSION THREAD

**Discussion by Ezgril** [stashapp/stash#5081] [General]
This is probably a stupid question but I'm interested in using stash on my smartphone and ipad, both running Android. I've been looking for a while and can't find anything about it. If not, is there any plan to make an Android version?

Thanks

**Comment by DogmaDragon** [comment]
Stash is a web app, but we only offer desktop binaries. You can run it on a desktop device and access it from other devices on the same network. All you need is a browser.

**Reply by Ezgril** [reply]
Thanks

**Reply by David-Maisonave** [reply]
One option would be to install Docker.
Here's step by step instructions for installing Docker on Android: https://gist.github.com/FreddieOliveira/efe850df7ff3951cb62d74bd770dce27

**Comment by aicynide** [comment]
Install termux and run
`pkg ins tur-repo && pkg ins stash`

**Reply by wilshere13** [reply]
How can i access the web
